core/workspace_svg.js

```diff
@@ -1203,6 +1281,11 @@ Blockly.WorkspaceSvg.prototype.showContextMenu_ = function(e) {
   redoOption.callback = this.undo.bind(this, true);
   menuOptions.push(redoOption);
 
+  // Option to add a workspace comment.
+  if (this.options.comments) {
```

Comment:
Are we sure we're ready for this bit?  IIUC, it is going to automatically enable workspace comments for anyone who has a category toolbox.
- do we want to automatically opt in folks with block comments to workspace comments?
- are they well tested/settled, especially with events and the xml format to turn them on now?

Reply:
No, it's not ready for that.  Removed, and we can manually add it back in to test/develop.